src/backtrace.h

```diff
@@ -200,6 +206,8 @@ void collect_backtrace(void) {
 #else
         fprintf(stderr, "Backtrace support is not available.\n");        
 #endif
+    } else if (0 == strcmp(method, "")) {
+        fprintf(stderr, "Backtrace support is not enabled. Set SHMEM_BACKTRACE to auto, execinfo, or gdb to enable.\n");        
```

Comment:
Better not to list acceptable values here, since it adds another place where we need to maintain this list.  I would shorten this to: `Set SHMEM_BACKTRACE for additional details`.  But, also, is it necessary to advertise the feature in this way?